src/m365/spo/commands/group/group-member-add.spec.ts

```diff
@@ -362,13 +362,13 @@ describe(commands.GROUP_MEMBER_ADD, () => {
         debug: true,
         webUrl: "https://contoso.sharepoint.com/sites/SiteA",
         groupName: "Contoso Site Owners",
-        email: "Alex.Wilber@contoso.com"
+        emails: "Alex.Wilber@contoso.com"
       }
     });
     assert(loggerLogSpy.calledWith(jsonSingleUser.UsersAddedToGroup));
   });
 
-  it('adds user to a SharePoint Group by groupId and aadGroupId (Debug)', async () => {
+  it('adds user to a SharePoint Group by groupId and aadGroupIds', async () => {
```

Comment:
why drop the `(Debug)` ? I guess we may leave that as this is what we also pass in this test